spectacles/validators/data_test.py

```diff
@@ -50,6 +50,16 @@ def validate(self) -> Dict[str, Any]:
             model_name = test["model_name"]
             explore_name = test["explore_name"]
             query_url_params = test["query_url_params"]
+            file_path = test["file"].split("/", 1)[1]
+            explore_url = (
+                f"{self.client.base_url}/explore/{model_name}"
+                f"/{explore_name}?{query_url_params}"
+            )
+            lookml_url = (
+                f"{self.client.base_url}/projects/{self.project.name}"
+                f"/files/{file_path}?line={test['line']}"
+            )
```

Comment:
Nice! Hadn't realized we had everything needed to return this.

Reply:
I think you had actually done the first version of this. I'm just pulling it up from the later loop.